test/dlc_tests/container_tests/bin/testSmdebug

```diff
@@ -58,7 +58,7 @@ if [[ $FRAMEWORK == "tensorflow" ]]; then
     # Later version of tensorflow-datasets
     # do not work with tensorflow 1.x
     pip install tensorflow-datasets==3.2.1
-    python -m pytest --assert=plain tests/${FRAMEWORK} \
+    pytest tests/${FRAMEWORK} \
```

Comment:
What was the reason for making this change?

Reply:
`python -m pytest --assert=plain tests/${FRAMEWORK} \` was intended to fix pytest's alleged flakyness, but there was no real reason to invoked it though `python -m`